attachmeta.py

```diff
@@ -0,0 +1,236 @@
+import findmeta
+import eyed3
+import requests
+
+
+def splitByFirstOccurence(string, character):
```

Comment:
Name your functions in snake case. This is not Java or JavaScript. Follow PEP8 :)

Reply:
Alright.
Here we go: https://github.com/fossdelhi/sync-music/commit/39337ba2c2c41faa71d1a46eb08a604a07052650